### ISSUE TITLE
Unable to reproduce results for SACN model

### ISSUE THREAD

**Issue by svjan5** [JD-AI-Research-Silicon-Valley/SACN#5]
Hi,
Using the default hyperparameters, I am unable to reproduce MRR score `.33` for ConvTransE model from the given code. Currently, I am getting around `.295` on the test set. Can you share the hyperparameters you used for training ConvTransE?

Thanks in advance

**Comment by chaoshangcs** [comment]
Hi, thank you so much for your question. The model is sensitive to the parameters. Here are the basic hyperparameters for your reference. Thanks! If you have any problems, please email me. Thanks!

CUDA_VISIBLE_DEVICES=0 python main.py model ConvTransE init_emb_size 100 dropout_rate 0.4 channels 50 lr 0.001 dataset FB15k-237 process True

**Comment by svjan5** [comment]
Hi @chaoshangcs,
I could reproduce `ConvTransE` results with my own implementation. However, I could not reproduce SACN results.
